lib/install/template.rb

```diff
@@ -1,31 +1,39 @@
 # Install webpacker
+puts "Copying webpack configuration files"
+directory "#{__dir__}/config/webpack", "config/webpack"
+
 puts "Creating javascript app source directory"
-directory "#{__dir__}/javascript", "app/javascript"
+directory "#{__dir__}/javascript", "#{Webpacker::Configuration.source}"
 
 puts "Copying binstubs"
 directory "#{__dir__}/bin", "bin"
 chmod "bin", 0755 & ~File.umask, verbose: false
 
-puts "Copying webpack core config and loaders"
-directory "#{__dir__}/config/webpack", "config/webpack"
-directory "#{__dir__}/config/loaders/core", "config/webpack/loaders"
+puts "Copying babel and postcss config"
+copy_file "#{__dir__}/examples/javascript/.babelrc", ".babelrc"
```

Comment:
Installing a .babelrc by default seems like a good idea to me, but it does mean that an enthusiastic user that tries to install React through Webpacker afterwards will get a conflict on it. Maybe we can improve the installation of React by not trying to replace the .babelrc but try to add the react preset to it?